llvm/lib/Target/AIE/AIEMachineScheduler.cpp

```diff
@@ -88,6 +88,10 @@ static cl::opt<bool> UseLoopHeuristics(
     "aie-loop-sched-heuristics", cl::init(true),
     cl::desc("Use special picking heuristics when scheduling a loop region"));
 
+static cl::opt<bool> IgnoreMemoryBankConflict(
+    "aie-ignore-bank-conflict", cl::init(false),
+    cl::desc("Ignore bank conflicts based on special heuristics"));
```

Comment:
nit: I like this `special heuristics` but I think we can go with just heuristics ;-).